openedx/adg/lms/applications/models.py

```diff
@@ -231,6 +231,19 @@ def business_lines_for_user(cls, user):
         """
         return BusinessLine.objects.all() if is_superuser_or_adg_admin(user) else cls.objects.filter(group__user=user)
 
+    @classmethod
+    def user_emails_for_business_line(cls, business_line_title):
+        """
```

Comment:
What's the reason for placing this method in model rather than manager?

Reply:
`BusinessLine.user_emails_for_business_line(line)` made more sense to me than `BusinessLine.objects.user_emails_for_business_line(line)`

Also, all the other similar methods were also written on the class so I thought to keep this here as well so we don't have to go to and fro.

Should I move it to the manager?